src/api/firebase.js

```diff
@@ -225,19 +234,16 @@ export async function uncheckItem(listPath, itemId) {
 	const itemDoc = await getDoc(itemDocRef);
 	const itemData = itemDoc.data();
 
+	const newTotalPurchases = itemData.totalPurchases - 1;
```

Comment:
This line can be removed completely and subbed for `totalPurchases: increment(-1)` on line 245. This would remove some code and keep consistency with the `updateItem` function above.